.github/ISSUE_TEMPLATE/z_release_major.md

```diff
@@ -18,35 +18,35 @@ assignees: ''
 
 When development is ready to begin one of the engineers should be nominated as a Release Engineer. They will be responsible for making sure the following tasks are completed though not necessarily completing them.
 
-- [ ] Set the version number in the `package.json` and `package-lock.json`, then submit a PR to `master` branch. The easiest way to do this is to use `npm --no-git-tag-version version <major|minor>`.
-- [ ] Ensure that issues from merged commits are closed and mapped to a milestone.
+- [ ] Checkout to a new `<major>.<minor>.<patch>-release` branch (eg 4.4.x-release) and set the version number in the `package.json` and `package-lock.json`. The easiest way to do this is to use `npm --no-git-tag-version version <major|minor>`. Once the version is updated, submit a PR to `master` branch.
+- [ ] Ensure that issues associated with merged commits to `master` are closed and mapped to a milestone.
```

Comment:
```suggestion
- [ ] Ensure that issues associated with commits merged to `master` since the last release are closed and mapped to the milestone.
```